new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -116,8 +214,6 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    */
   @Test
```

Comment:
Delete the Order annotation from all method, if we do not need Order in the class

Reply:
done